extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -155,6 +147,8 @@ export class Bind {
       this.setStatePromise_ = this.initializePromise_.then(() => {
         user().fine(TAG, 'State updated; re-evaluating expressions...');
         return this.digest_();
+      }).then(() => {
+        this.dispatchEventForTesting_('amp:bind:setState');
```

Comment:
Nit: This adds an extra tick to production code (even if this method does nothing). Maybe test `getMode` before adding this `#then` block?